404.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="resources/css/style.css">
+  <link rel="stylesheet" type="text/css" href="resources/css/responsive.css">
+  <title>Rmedia</title>
+
+  <!-- Favicons -->
+  <link rel="apple-touch-icon" sizes="180x180" href="/resources/favicons/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/resources/favicons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/resources/favicons/favicon-16x16.png">
+  <link rel="manifest" href="/resources/favicons/site.webmanifest">
+  <link rel="mask-icon" href="/resources/favicons/safari-pinned-tab.svg" color="#5bbad5">
+  <link rel="shortcut icon" href="/resources/favicons/favicon.ico">
+  <meta name="msapplication-TileColor" content="#da532c">
+  <meta name="msapplication-config" content="/resources/favicons/browserconfig.xml">
+  <meta name="theme-color" content="#ffffff">
+</head>
+<body>
+  
+  <header>
+    <div class="container">
+      <h1><a href="#"><img src="resources/images/logo.jpg" alt="Rmedia"></a></h1>
+      <input type="search" name="search" id="search" placeholder="search...">
+    </div>
+  </header>
+
+  <section class="content">
+
+    <div class="container">
+
+      <div class="content-section">
+        <section class="left-section">
+          <nav class="navigation">
+            <ul id="nav-menues">
+              <!-- <li><a href="#">Home</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Service</a></li>
+              <li><a href="#">Contact</a></li> -->
+            </ul>
+          </nav>
+        </section>
+        <section class="middle-section">
+          <h2 class="not-found">404</h2>
+          <p class="not-found-text">Page Not Found</p>
+          <img src="resources/images/404.jpg" alt="not found image" class="not-found-image">
+        </section>
+        
+        <section class="right-section">
+          <h3 class="post-heading">Latest Post</h3>
+          <ul class="posts">  
+            <!-- <li><a href="#">
```

Comment:
Remove commented code

Reply:
Please fix the style break issues in smaller resolution.

Reply:
Commented code removed